CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,153 @@
+# Release notes for v2.2.0
+
+[Documentation](https://kubernetes-csi.github.io)
+# Changelog since v2.1.0
+
+## Changes by Kind
+
+### API Change
+
+- If the CSI driver reports `MaximumVolumeSize`, that value is also published in the `CSIStorageCapacity` objects. The v1beta1 storage API from Kubernetes >= 1.21 is required. ([#584](https://github.com/kubernetes-csi/external-provisioner/pull/584), [@pohly](https://github.com/pohly))
+
+### Feature
+
+- After volume creation or deletion, those CSIStorageCapacity objects most likely affected by that get refreshed sooner than the others. ([#586](https://github.com/kubernetes-csi/external-provisioner/pull/586), [@pohly](https://github.com/pohly))
+- New metrics data (storage capacity tracking, process and Go runtime, work queues, leaderelection) ([#579](https://github.com/kubernetes-csi/external-provisioner/pull/579), [@pohly](https://github.com/pohly))
```

Comment:
There is https://github.com/kubernetes-csi/docs/issues/339 for metrics documentation. This is a bigger issue because there's basically no documentation at all at the moment and I wasn't sure where to start when adding the new ones.